src/garage/torch/modules/__init__.py

```diff
@@ -1,4 +1,5 @@
 """Pytorch modules."""
 from garage.torch.modules.mlp_module import MLPModule
+from garage.torch.modules.gaussian_mlp_module import GaussianMLPModule, GaussianMLPAdaptiveModule, GaussianMLPSharedModule
 
-__all__ = ['MLPModule']
+__all__ = ['MLPModule', 'GaussianMLPModule', 'GaussianMLPAdaptiveModule', 'GaussianMLPSharedModule']
```

Comment:
I suggest `GaussianMLPIndependentStdModule` and `GaussianMLPTwoHeadedModule`

Reply:
fixed